package.json

```diff
@@ -24,15 +24,18 @@
         "args-parser": "^1.3.0",
         "axios": "^0.21.1",
         "bcrypt": "^5.0.1",
+        "bcryptjs": "^2.4.3",
```

Comment:
What is the different between bcrypt and bcryptjs?

Reply:
As above, I suspect this has been pulled in by something, let me take a look and see if I can remove it.